packages/generator-slimer/generators/test/index.js

```diff
@@ -1,8 +1,8 @@
 'use strict';
 const Generator = require('../../lib/Generator');
 
-// "test": "NODE_ENV=testing c8 --all --check-coverage --reporter text --reporter cobertura mocha './test/**/*.test.js'",
-const testScript = 'NODE_ENV=testing c8 --all --check-coverage --reporter text --reporter cobertura mocha \'./test/**/*.test.js\'';
+// "test": "NODE_ENV=testing c8 --all --check-coverage --lines 100 --functions 100 --branches 100 --reporter text --reporter cobertura mocha './test/**/*.test.js'",
+const testScript = 'NODE_ENV=testing c8 --all --check-coverage --lines 100 --functions 100 --branches 100 --reporter text --reporter cobertura mocha \'./test/**/*.test.js\'';
```

Comment:
Is there a reason for not including `--statements 100` too? If not, we should be able to use `--100` as per https://github.com/bcoe/c8#checking-coverage 🕺🏻

Reply:
My thinking was we'd might need to modify these separately. But then it's an untested assumption! Let's go with the shorter `--100` and see if we get any friction with this approach 👌

Reply:
Good point! Maybe the friction will encourage people not to reduce the thresholds haha

Reply:
that's a good perspective. pain of figuring out how to reduce the threshold > writing the damn test 😂